BedrockServer.cpp

```diff
@@ -491,9 +494,12 @@ void BedrockServer::worker(SData& args,
 {
     SInitialize("worker" + to_string(threadId));
 
+    // Check if the synchronous flag was passed, if not set to false and let the SQLite default get used
+    string synchronous = args.isSet("-synchronous") ? args["-synchronous"] : "false";
+
     // We pass `0` as the checkpoint size to disable checkpointing from workers. This can be a slow operation, and we
     // don't want workers to be able to block the sync thread while it happens.
-    SQLite db(args["-db"], args.calc("-cacheSize"), 0, args.calc("-maxJournalSize"), threadId, threadCount - 1);
+    SQLite db(args["-db"], args.calc("-cacheSize"), 0, args.calc("-maxJournalSize"), threadId, threadCount - 1, synchronous);
```

Comment:
Just pass `args["-synchronous"]` and let it be blank if not set (same as above).